sbt-bridge/resources/META-INF/services/xsbti.compile.CompilerInterface2

```diff
@@ -0,0 +1 @@
+xsbt.CompilerBridge
```

Comment:
The point of the ServiceLoader approach is that we can use any class name we want as long as we write it in this file, so we should take advantage of that and call this `dotty.tools.zincBridge.CompilerBridge` or something like that.

Reply:
I moved all the classes used by the new interface to the `dotty.tools.xsbt` package.